### ISSUE TITLE
Web interface ?

### ISSUE THREAD

**Issue by AntoineTurmel** [dfirebaugh/um3-wooter#2]
Nice project :) I'm looking into doing a similar thing for Discord.
What's the web interface for https://ultimaker.hackrva.org/ ?

**Comment by dfirebaugh** [comment]
Thank you.
The web dashboard is something I wrote a long time ago. 

https://github.com/dfirebaugh/um3-react-dashboard

Here is an earlier version that doesn't use any frameworks
https://github.com/dfirebaugh/Ultimaker3-Web-Dashboard


I made that before Ulitmaker had the nice web interface.  I'm not sure if the default ultimaker web interface should be exposed to the web though.


Let me know if you have any questions while you make your thing.  The slack thing was fairly straight forward.

**Comment by AntoineTurmel** [comment]
Yeah, I was think of having the bot on a raspberry pi connected on our network for example.
And I was thinking also of adding feature like !photo -> post a photo during the print and also filtering out private projects.
